Docs/Specifics/Choreo.md

```diff
@@ -24,6 +25,27 @@ The Choreo GUI communicates with Sleipnr and receives results back (as Sleipnir
 
 The official Choreo [usage docs](https://sleipnirgroup.github.io/Choreo/usage/editing-paths/) give the best instructions on how to use the Choreo GUI app for creating these paths.
 
+## Why Choreo
+
+### Intro to Numerical Optimization
+
+Choreo is a new approach to creating auto trajectories by using numerical optimization, namely Convex Optimization. Convex Optimization with respect to time constraints applies to minimizing the time it takes to travel a path. A convex function simply is a function that curves downward, like an upward parabola. The constraints (independent variables) axis would be changed to affect the time (dependent variables) axis. It's the solver's job to try as hard as possible to minimize the time taken for a path, which in turn allows robotics teams to score more points in the auto period.
```

Comment:
"A convex function simply is a function that curves downward, like an upward parabola."
not exactly, its a function where you can draw a line between any point in the function to any other point in the function without intersecting the function.